reflex/vars.py

```diff
@@ -201,7 +219,10 @@ def dict(self) -> dict:
             "state": self.state,
             "interpolations": list(self.interpolations),
             "imports": {
-                lib: [import_var.dict() for import_var in import_vars]
+                lib: [
+                    import_var.dict() if isinstance(import_var, ImportVar) else {}
```

Comment:
is this for the type checker? when would `import_var` not be an `ImportVar` here? what else could it be?

Reply:
Will remove, it was needed in the middle of dev because the import_var could be a str, but now it's parsed properly ahead of time so it won't happens.